docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -71,3 +71,16 @@ items:
       kubernetes.io/hostname: ocp-worker-0
       node-role.kubernetes.io/worker: ""
 ```
+
+[WARNING]
+
+In IPv6, {OpenShift} worker nodes need a /64 prefix allocation due to OVN
+limitations (RFC 4291). For dynamic IPv6 configuration this needs to be changed
+on the Router Advertisement settings. In case you want to use manual
+configuration for IPv6, define a similar CR to the one seen above that enables
+and defines an IPv6 address and disables IPv4.  Considering that these
+{Openshift} worker nodes are serving as what we called Controllers in
+{OpenstackPreviousInstaller}, where this constraint for the /64 prefix didn't
+exist, there is a chance that your previous {OpenStackShort} Control Plane
+network does not have a size enough to allocate these networks.  If that is the
+case, allocate a prefix wide enough to fit them.
```

Comment:
```suggestion
[WARNING]
In IPv6, {OpenShift} worker nodes need a `/64` prefix allocation due to OVN limitations (RFC 4291). For dynamic IPv6 configuration, you need to change the prefix allocation on the Router Advertisement settings. If you want to use manual configuration for IPv6, define a similar CR to the `NodeNetworkConfigurationPolicy` CR example in this procedure, and define an IPv6 address and disable IPv4.  Because the constraint for the `/64` prefix did not exist in {OpenstackPreviousInstaller}, your previous {OpenStackShort} control plane network might not have enough capacity to allocate these networks. If that is the case, allocate a prefix large enough to fit them.
```

Reply:
@elvgarrui Can you please ensure that I did not change the intended meaning of this note?
Also, would it make sense to add an example value in the last sentence? E.g., "If that is the case, allocate a prefix large enough to fit them, for example, 110."

Reply:
I updated it now, let me know what you think.